include/swift/IDE/SyntaxModel.h

```diff
@@ -146,6 +147,7 @@ struct SyntaxStructureNode {
     case SyntaxStructureKind::StaticVariable:
     case SyntaxStructureKind::ClassVariable:
     case SyntaxStructureKind::Parameter:
+    case SyntaxStructureKind::Subscript:
```

Comment:
I'm not sure if this is the proper solution or if it's better to:
1. Rename this function
2. Whenever using `isVariable` also check if `Kind == Subscript`